src/toil/job.py

```diff
@@ -982,9 +982,9 @@ def deleteLocalFile(self, fileStoreID):
                             os.remove(cachedFile)
                             cacheInfo.cached -= fileSize
                     self.logToMaster('Successfully deleted cached copy of file with ID '
-                                     '\'%s\'.' % fileStoreID)
+                                     '\'%s\'.' % fileStoreID, logging.DEBUG)
                 self.logToMaster('Successfully deleted local copies of file with ID '
-                                 '\'%s\'.' % fileStoreID)
+                                 '\'%s\'.' % fileStoreID, logging.DEBUG)
 
```

Comment:
I agree with this change, I would go further and turn these into standard log messages. @arkal - in the case that the job succeeds do you really want to see this message? Seems like a lot of low level information. The current design for "global" logging messages is to convey them all back to the leader. If you want to persist this behavior we should filter log to masters on the worker by the log level to prevent excessive log passing during a production run.